src/types.rs

```diff
@@ -1,4 +1,6 @@
-use primitive_types::U256;
+use fixed::types::U89F39 as F;
 
 pub type Balance = u128;
-pub type Balance256 = U256;
+pub type FixedBalance = F;
```

Comment:
yes, but helps if need to change the type again. I was experimenting with multiple precisions, and this way I had to change only in one place, rather than two.